docs/tutorials/plots.rst

```diff
@@ -445,6 +447,43 @@ any plotting function:
     plt.show()
 
 
+Additional Options
+++++++++++++++++++
+
+You can also shade the background according to the darkness of the sky (light
+shading for 0 degree twilight, darker shading for -18 degree twilight) with
+the `brightness_shading` keyword, and display additional y-axis ticks on the
```

Comment:
Double backticks around kwargs otherwise sphinx tries to linkify them. (below for ``latitude_yaxis``, too)